### PR TITLE
Add --all option in support bundle to collect all logs including stacktrace, binlogs and coredumps

### PR DESCRIPTION
**Pull request by rohit-k-dwivedi**
Signed-off-by: Rohit Dwivedi <rohit.k.dwivedi@seagate.com>

JIRA: https://jts.seagate.com/browse/CORTX-29690
# Problem Statement
- users won't be able to remember all the options of support bundle, so adding `--all`
 option to set all flags to true for collecting all logs
# Design
-  added `--all` option to collect  stacktrace, binlogs and coredumps with single option

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [x] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [x] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: src/rgw/support/rgw_support_bundle [hunk 1]
```diff
@@ -157,6 +157,9 @@ class RGWSupportBundle:
             help="Include/Exclude stacktrace, Default = False")
         parser.add_argument('--modules', dest='modules',
             help="list of components & services to generate support bundle.")
+        parser.add_argument('--all', type=RGWSupportBundle.str2bool, default=False,
+            help="Include/Exclude all logs[stacktrace, Coredumps, Binary], Default = False")
```
Comment (cdeshmukh) [inline src/rgw/support/rgw_support_bundle:161]: help="Include all debug data, including logs, config, stack traces, core dumps, binaries, etc, possibly resulting in much HEAVIER support bundle], Default = False")

Comment (cdeshmukh) [reply src/rgw/support/rgw_support_bundle:161]: @rohit-k-dwivedi add ticket id to commit msg

**Comment by cdeshmukh** [comment]
We would need similar '-all' support in CORTX support bundle code too, to propagate to component support bundles

File: src/rgw/support/rgw_support_bundle (hunk 1 above)
Comment (rohit-k-dwivedi) [reply src/rgw/support/rgw_support_bundle]: done

**Review by cdeshmukh — APPROVED** [review]

**Review by sachitanands — APPROVED** [review]
looks good

**Review by suryakumar1024 — APPROVED** [review]
LGTM

**Review by cdeshmukh — APPROVED** [review]

**Pull request merged**